lib/coins/btc.js

```diff
@@ -24,7 +25,9 @@ function parseUrl (network, url) {
 
   if (!validate(network, address)) throw new Error('Invalid address')
 
-  return address
+  return isBech32Address(address, bech32Opts)
```

Comment:
This should be generalized more. We need a new ``formatAddress`` function that takes an address and lowercases it when necessary.